### ISSUE TITLE
Angry

### ISSUE THREAD

**Issue by Timed-kitty-cat** [Timed-kitty-cat/Timed-kitty-cat#19]
# Main
You know \*removed because you could misclick if not careful, be warned!\*? It's DISGUSTING. ~~Also, a REVIVAL?~~ The revival was deleted because it could not be. Ranting:
> If that was the case, Addison would be locked in the Astral Tower as another user said that the Astral Warden is- oh wait why am I pointlessly rambling about Gale and Addison?

See https://github.com/Timed-kitty-cat/Timed-kitty-cat/issues/19#issuecomment-878636239 if you want link 
## Stuff
\*insert lavender town theme here\* Yeah I'm confused about why it exists, you?

**Comment by Time-dragon** [comment]
ah well, search google for gale prodigy for it

**Comment by Time-dragon** [comment]
You CLEARLY misunderstood, Google. Someone searching for images of "gale prodigy" should NOT get [this](https://prodigy-math-game.fandom.com/f/p/4400000000000171104).

**Comment by Time-dragon** [comment]
Banjo

**Comment by Time-dragon** [comment]
Banjo

**Comment by Time-dragon** [comment]
<img width="378" alt="kitty" src="https://user-images.githubusercontent.com/82837930/125091674-a0133500-e09e-11eb-911f-ba24a6bc306c.png">
cat says locked

**Comment by Timed-kitty-cat** [comment]
In my opinio-

**Comment by Timed-kitty-cat** [comment]
Time to see if DJMORON is unba-

**Comment by Timed-kitty-cat** [comment]
-nned.

**Comment by Timed-kitty-cat** [comment]
At least she does not use GitHub. That's a relief.

**Comment by Timed-kitty-cat** [comment]
Anyway, paradox:

https://user-images.githubusercontent.com/82837930/125207749-f9749300-e25b-11eb-8989-9e882b3bbf7e.mp4

**Comment by Timed-kitty-cat** [comment]
wresfdxkfdx,vbxdfz.gbsdfbszcbxkvn,z patch

**Comment by Timed-kitty-cat** [comment]
one thing N O

**Comment by Timed-kitty-cat** [comment]
misclicked wah

**Comment by Timed-kitty-cat** [comment]
BTW if you need the link it's https://prodigy-math-game.fandom.com/f/p/4400000000000171104

**Comment by Timed-kitty-cat** [comment]
now regret \*trademark sign\*

**Comment by Timed-kitty-cat** [comment]
LET THE LAKE FEEL THE ROLLING THUNDER

**Comment by Timed-kitty-cat** [comment]
i mad

**Comment by Timed-kitty-cat** [comment]
Give me 20 reasons to click that

**Comment by Timed-kitty-cat** [comment]
> Give me 20 reasons to click that

1. NO

**Comment by Timed-kitty-cat** [comment]
no u

**Comment by Timed-kitty-cat** [comment]
\*Insert the Giga Bell ringing here\* Grab it and calm down!

**Comment by Timed-kitty-cat** [comment]
rake bile

**Comment by Timed-kitty-cat** [comment]
hold on do i steal the giga bells oh wait i do now here take this laptop for ruining my day by making me click the wrong link and i had to close the tab panickedly and deep breathe

**Comment by Timed-kitty-cat** [comment]
ok now i h-

**Comment by Timed-kitty-cat** [comment]
\*awkward silence\*

**Comment by Timed-kitty-cat** [comment]
-ave C H O R T L E S

**Comment by Timed-kitty-cat** [comment]
Yes I see Google has the link. But only a P I E C E of it. The part we humans laugh at.

**Comment by Timed-kitty-cat** [comment]
\*punches Google\*

**Comment by Timed-kitty-cat** [comment]
I also have F U R Y

**Comment by CubeyTheCube** [comment]
this is so sad can we get an f

**Comment by Timed-kitty-cat** [comment]
@CubeyTheCube yes we can

**Comment by Timed-kitty-cat** [comment]
hmm

**Comment by Timed-kitty-cat** [comment]
Hmm it still exists how cringy

**Comment by Timed-kitty-cat** [comment]
uh it s t a l e
